src/lib_python.rs

```diff
@@ -0,0 +1,15 @@
+// TODO: remove before 1.0.0
+#![allow(dead_code)]
+
+use pyo3::prelude::*;
+
+mod core;
+mod error;
+
+pub use crate::core::matter::Matter; 
```

Comment:
I don't know that this `pub` is required, now that I'm reading it. I figured it would be when I started but the way the class is added makes me wonder.

Reply:
Removed, was unnecessary.